config_indi.h

```diff
@@ -53,6 +53,7 @@ enum IndiDevType
     INDI_TYPE_MOUNT,
     INDI_TYPE_AUX_MOUNT,
     INDI_TYPE_AO,
+    INDI_TYPE_ROTATOR
```

Comment:
minor, to simplify future diffs:
```suggestion
    INDI_TYPE_ROTATOR,
```